TrainingTask/Task2Hunter/Hunting.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TrainingTask.Task2Hunter
+{
+    public class Hunting
+    {
+        public Hunter Hunter { get; set; }
+
+        public Rabbit Rabbit { get; set; }
+
+        public Hunting(Hunter hunter, Rabbit rabbit)
+        {
+            this.Hunter = hunter;
+            this.Rabbit = rabbit;
+
+            this.Rabbit.EventNewLocation += this.OnEventNewLocationInvoked;
```

Comment:
According to the general logic, hunter is hunting so he should be subscribed to rabbit.